setup.py

```diff
@@ -1562,17 +1552,17 @@ class db_found(Exception): pass
                                       expat_depends,
                                   ))
         else:
-            missing.append('_elementtree')
+            self.missing.append('_elementtree')
 
         # Hye-Shik Chang's CJKCodecs modules.
-        exts.append(Extension('_multibytecodec',
+        self.add_ext(Extension('_multibytecodec',
                               ['cjkcodecs/multibytecodec.c']))
         for loc in ('kr', 'jp', 'cn', 'tw', 'hk', 'iso2022'):
-            exts.append(Extension('_codecs_%s' % loc,
+            self.add_ext(Extension('_codecs_%s' % loc,
                                   ['cjkcodecs/_codecs_%s.c' % loc]))
 
         # Stefan Krah's _decimal module
-        exts.append(self._decimal_ext())
+        self._detect_decimal()
```

Comment:
You don't need to add to `self.add_ext` here?

Reply:
See the commit message, I modified _detect_decimal() to call directly add_ext(). Previously, it wasn't possible since 'exts' was a local variable.

Reply:
arfff sorry